go/vt/sqlparser/testdata/select_cases.txt

```diff
@@ -12913,8 +12913,8 @@ END
 INPUT
 select collation(cast('a' as char(2))), collation(cast('a' as char(2) binary));
 END
-ERROR
-syntax error at position 77 near 'binary'
+OUTPUT
+select collation(convert('a', char(2))), collation(convert('a', char(2) binary)) from dual
```

Comment:
Yay, we now can handle these cases that were before marked as a syntax error :smile:. They also work on MySQL itself.